frontend/lib/shared/routes/routes.dart

```diff
@@ -3,26 +3,48 @@ import 'package:utm_marketplace/login/view/login.view.dart';
 import 'package:utm_marketplace/profile/view/profile.view.dart';
 import 'package:utm_marketplace/item_listing/view/listing.view.dart';
 
-// TODO: These are temporary imports for testing purposes, remove when no longer needed
-import 'package:utm_marketplace/temp_view/view/temp_view.view.dart';
-import 'package:utm_marketplace/temp_view/view_models/temp_model.viewmodel.dart';
-import 'package:utm_marketplace/home/view/home.view.dart';
+import 'package:utm_marketplace/shared/components/shell/shell.layout.dart';
+import 'package:utm_marketplace/messages/view/messages.view.dart';
+import 'package:utm_marketplace/notifications/view/notifications.view.dart';
+import 'package:utm_marketplace/menu/view/menu.view.dart';
 
 // Define the router as a top-level global variable
-final GoRouter router = GoRouter(
+final router = GoRouter(
   initialLocation: '/login',
   routes: [
     GoRoute(
       path: '/login',
       builder: (context, state) => const Login(),
     ),
     GoRoute(
-      path: '/temp_view',
-      builder: (context, state) =>
-          ModelView(model: Model(attribute1: 'Attribute 1', attribute2: 2)),
+      path: '/item_listing',
```

Comment:
I think this route should be removed. There wouldn't be a case where the item listing / marketplace page is going to be shown without the bottom navbar.